src/MicroOrm.Dapper.Repositories/DapperRepository.Insert.cs

```diff
@@ -84,7 +84,7 @@ public virtual async Task<bool> InsertAsync(TEntity instance, IDbTransaction tra
         private bool SetValue(long newId, TEntity instance)
         {
             var added = newId > 0;
-            if (added)
+            if (added && SqlGenerator.IsIdentity)
```

Comment:
Why change the logic in this PR? Raise another PR if you want to change logic or fix bugs

Reply:
`SqlGenerator.IdentitySqlProperty` maybe null.

Reply:
ok, not change here